src/Avalonia.Base/Input/IFocusScope.cs

```diff
@@ -1,6 +0,0 @@
-namespace Avalonia.Input
-{
-    public interface IFocusScope
```

Comment:
I remember IFocusScope was added only to support Popups, and now same problem was resolved with internal Popup.ManagesFocus (as I understood). But do we have any focus scoping now in this PR, to isolate focus temporary and restore previous focus when scope removed?  @grokys